### PR TITLE
Update CHANGELOG with warnings about TEA deployment

### PR DESCRIPTION
**Pull request by markdboyd**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,13 +9,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 ### BREAKING CHANGES
 
 - **CUMULUS-2255** - Cumulus has upgraded its supported version of Terraform from **0.12.12** to **0.13.6**.
+  - Because of a change in Terraform v0.13 where `aws_lambda_invocation` data resources are evaluated during the plan/refresh phase of deployment, you may see deployment errors if you add/update the `public`/`protected` buckets in the `buckets` variable for your deployment. The error would look like this:
+
+  ```shell
+  Error: Lambda function (<prefix>-tf-TeaCache) returned error: ({"errorType":"HTTPError","errorMessage":"Response code 404 (Not Found)","trace":["HTTPError: Response code 404 (Not Found)","    at Request.<anonymous> (/var/task/node_modules/got/dist/source/as-promise/index.js:117:42)","    at processTicksAndRejections (internal/process/task_queues.js:97:5)"]})
+  ```
+
+  For now, the workaround if you see this issue is to manually delete the `<prefix>-tf-TeaCache`, which will fix the race condition issue and allow you to deploy.
```
Comment (botanical) [inline CHANGELOG.md:26]:
```suggestion
  For now, the workaround if you see this issue is to manually delete the `<prefix>-tf-TeaCache`, which will fix the race condition issue and allow you to successfully deploy.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,13 +9,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 
 - **CUMULUS-2350**
   - If the  `/s3credentialsREADME`, does not appear to be working after deployment, [manual redeployment](https://docs.aws.amazon.com/apigateway/latest/developerguide/how-to-deploy-api-with-console.html) of the API-gateway stage may be necessary to finish the deployment.
 
 ### BREAKING CHANGES
 
 - **CUMULUS-2255** - Cumulus has upgraded its supported version of Terraform from **0.12.12** to **0.13.6**.
+  - Because of a change in Terraform v0.13 where `aws_lambda_invocation` data resources are evaluated during the plan/refresh phase of deployment, you may see deployment errors if you add/update the `public`/`protected` buckets in the `buckets` variable for your deployment. The error would look like this:
+
+  ```shell
```
Comment (botanical) [inline CHANGELOG.md:22]: Does "```shell" work differently in markdown? It seems to render the code as a single long line.

Comment (markdboyd) [reply CHANGELOG.md:22]: I can add some line breaks

**Review by botanical — APPROVED** [review]
Thanks for updating this!

**Review by Jkovarik — CHANGES_REQUESTED** [review]
I have a fix for this.  We should *not* merge this PR.

**Comment by markdboyd** [comment]
Superseded by #2083

**Pull request closed without merging**